backend/indexer.py

```diff
@@ -629,8 +629,8 @@ def index_meta(self):
         new_db = current_db
         print "Reindexing into database %s" % new_db
         print "Note that this is not perfect! Do not use in production."
-        redis_conn.select(new_db)
-        redis_conn.set("hold", "1")
+        # redis_conn.select(new_db)
+        # redis_conn.set("hold", "1")
```

Comment:
I'd argue that with dockerisation there is zero reason to have the db flipping mechanism and we should just drop db selection, the magic key that tells us what the live database is, and so remove `flip_db` entirely.

Reply:
Makes sense. I suppose we can always revert it if we want to bring back the old deployment system.

I wonder what else we can delete? 😈

Reply:
Chef! 👨🏼‍🍳🔥